web/templates/index.html

```diff
@@ -42,8 +42,6 @@
                     <div>
                         <a href="https://github.com/ArduPilot/CustomBuild/issues/new" class="btn btn-outline-light me-1"><i
                                 class="bi bi-github me-2"></i>Report an issue</a>
-                        <a href="/builds" class="btn btn-outline-light me-1"><i class="bi bi-folder2-open me-2"></i>Go to
```

Comment:
Curious why this is going...

Reply:
Well, given @rmackay9 was wondering why we needed that at all the other day, I think we can do without it.